packages/thumbprint-tokens/src/index.js

```diff
@@ -2,12 +2,34 @@
 const path = require('path');
 const glob = require('glob');
 const fse = require('fs-extra');
+const JSZip = require('jszip');
 const handlebars = require('handlebars');
 
 const outputs = [
     { slug: 'javascript-cjs', distName: 'index.js' },
     { slug: 'javascript-es', distName: 'index.js' },
     { slug: 'scss', distName: '_index.scss' },
+    {
+        slug: 'ios',
+        distName: 'index.swift',
```

Comment:
Any objections to renaming this to `ThumbprintTokens.swift`? More standard in Swift naming conventions than `index`